### PR TITLE
Remove unused argument, and unused property

### PR DESCRIPTION
**Pull request by dunglas**
### Changes

Remove dead code.

### Testing

Please describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors. 

[x] This change adds test coverage

[x] This change has been tested on the latest version of Symfony

### Checklist

[x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)

[x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)

[x] All existing and new tests complete without errors

### CODE REVIEW

**Comment by darthf1** [comment]
ping @lbalmaceda @joshcanhelp

File: src/Security/Auth0Service.php [hunk 1]
```diff
@@ -58,7 +56,7 @@ public function __construct($api_secret, $domain, $api_identifier, $authorized_i
      *
      * @return array info as described in https://auth0.com/docs/users/concepts/overview-user-profile
      */
-    public function getUserProfileByA0UID($jwt, $a0UID)
```
Comment (lbalmaceda) [inline src/Security/Auth0Service.php:61]: wouldn't those calling this function with 2 args now receive an error?

Comment (dunglas) [reply src/Security/Auth0Service.php:61]: No, PHP allows to pass more arguments to a function. They are just ignored (and can be accessed using `func_get_args()`).

**Comment by Widcket** [comment]
This is now out in [3.4.0](https://packagist.org/packages/auth0/jwt-auth-bundle#3.4.0).

**Pull request merged**